docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -17,45 +12,34 @@ image::detections/images/alert-page.png[]
 === View and filter detection alerts
 The Alerts page offers various ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
 
-* Filter for a specific rule in the KQL bar (for example,
-`kibana.alert.rule.name :"SSH (Secure Shell) from the Internet"`).
-
-+
-NOTE: KQL autocomplete for `.alerts-security.alerts-*` indices is available on the
-*Alerts* and *Rule details* pages, and in Timeline when either `All` or
-`Detection alerts` is selected.
```

Comment:
Trimmed down this note, because it also talked about a lot more than just the Alerts page. We might want to make sure the Rule details and Timeline docs mention KQL autocomplete.